src/ripple/test/jtx/impl/Env_test.cpp

```diff
@@ -561,6 +561,21 @@ class Env_test : public beast::unit_test::suite
                 );
     }
 
+    void testSig()
+    {
+        using namespace jtx;
+        Env env(*this);
+
+        env.fund(XRP(10000), "alice");
+        auto const baseFee = env.app().config().FEE_DEFAULT;
+        std::uint32_t const aliceSeq = env.seq ("alice");
+        
+        Json::Value jsonNoop = env.json (
+            noop ("alice"), fee(baseFee), seq(aliceSeq), sig("alice"));
+        JTx jt = env.jt (jsonNoop);
+        env (jt);
```

Comment:
@ximinez, thanks for the review.  You're pointing out that, at the very least, this test case needs comments and possibly a better name.  This particular test case is intended to test whether jtx can re-sign a transaction that has already been signed.  Without the change in utility.cpp this test fails, but it passes with the change.

Without the change in utility.cpp five of your tests fail.  With the change in utility.cpp all of your tests pass.  I haven't looked at your tests in detail, but it seems like passing is the correct behavior.  Am I understanding your tests correctly?

I'll commit a FOLD that includes comments.  You can tell me whether the comments are sufficient.